src/util/mpir_pmi.c

```diff
@@ -828,3 +853,69 @@ static void free_pmi_keyvals(INFO_TYPE ** kv, int size, int *counts)
         MPL_free(counts);
     }
 }
+
+static int parse_coord_file(const char *filename)
```

Comment:
Add a comment explaining the format, e.g.
```
/* example coord file */
rank: port_id:switch_id:group_id:
0:0 0 0
1:1 0 0
....
*/


```

Reply:
Added the comment